Update Readme.md (#1)
DevExpress-Examples/asp-net-web-forms-file-manager-list-custom-file-system-provider: Readme.md

<!-- default badges list -->
[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E5024)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->

> # File Manager for ASP.NET Web Forms - How to implement a custom file system provider for List data source

Reviewer: ```suggestion
# File Manager for ASP.NET Web Forms - How to implement a custom file system provider for a List data source
```

<!-- run online -->
**[[Run Online]](https://codecentral.devexpress.com/e5024/)**
<!-- run online end -->

> This example demonstrates how to implement a custom file system provider that bounds [ASPxFileManager](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxFileManager) to a List data source. To persist data between requests, list is stored in the Session.

Reviewer: ```suggestion
This example demonstrates how to implement a custom file system provider that bounds [ASPxFileManager](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxFileManager) to a List data source. To persist data between requests, the list is stored in the Session.
```

## Files to Review

* [Default.aspx](./CS/WebApplication2/Default.aspx) (VB: [Default.aspx](./VB/WebApplication2/Default.aspx))
* [FileStreamProvider.cs](./CS/WebApplication2/FileStreamProvider.cs) (VB: [FileStreamProvider.vb](./VB/WebApplication2/FileStreamProvider.vb))

## Documentation

* [Custom File System Provider](https://docs.devexpress.com/AspNet/9907/components/file-management/file-manager/concepts/file-system-providers/custom-file-system-provider)

## More Examples

> * [File Manager for ASP.NET Web Forms - How to implement a custom file system provider for LINQ to SQL data source](https://github.com/DevExpress-Examples/asp-net-web-forms-filemanager-linq-to-sql-file-system-provider)

Reviewer: Do you mean

```suggestion
* [File Manager for ASP.NET Web Forms - How to implement a custom file system provider for LINQ and SQL data sources](https://github.com/DevExpress-Examples/asp-net-web-forms-filemanager-linq-to-sql-file-system-provider)
```
